rust/src/lib.rs

```diff
@@ -34,54 +48,66 @@ impl GraphWrapper {
         }
     }
 
-    pub fn get_modules(&self) -> FxHashSet<String> {
+    pub fn get_modules(&self) -> HashSet<String> {
```

Comment:
What's the reason for this change?

Reply:
Interesting I forgot about this one. I thought it was necessary since `FxHashSet` doesn't implement `IntoPy`, but presumably your code was working before so I'm now unsure 🤔